tests/wpunit/Tribe/Utils/Post_ID_HelperTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Tribe\Utils;
+
+use \Tribe__Main as Main;
+
+class Post_ID_HelperTest extends \Codeception\TestCase\WPTestCase {
+
+    /**
+     * dataProvider callback for getting a test event.
+     *
+     * @since TBD
+     */
+    public function get_sample_event() {
+        return [
+            'WP_Post' => [ $this->factory()->post->create_and_get( [ 'post_title' => 'Sample Event' ] ) ]
+        ];
+    }
+
+    /**
+     * @test When passing a WP_Post object, get the ID from it.
+     *
+     * @since TBD
+     *
+     * @param WP_Post $event_obj
+     *
+     * @dataProvider get_sample_event
+     */
+    public function it_should_return_post_ids_when_passed_post_objects( \WP_Post $event_obj ) {
```

Comment:
also pass `use \WP_Post;`

Reply:
done in the new PR → https://github.com/moderntribe/tribe-common/pull/964